webdriver/tests/bidi/browsing_context/reload/reload.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+pytestmark = pytest.mark.asyncio
+
+PNG_BLACK_DOT = "/webdriver/tests/bidi/browsing_context/support/black_dot.png"
+
+
+async def test_return_value(bidi_session, inline, new_tab):
+    url = inline("<div>foo</div>")
+    await bidi_session.browsing_context.navigate(context=new_tab["context"],
+                                                 url=url)
+
+    result = await bidi_session.browsing_context.reload(
+        context=new_tab["context"])
+    assert result == {}
+
+
+@pytest.mark.parametrize(
+    "initial_url",
+    [
+        "about:blank",
+        "https://example.com/#foo",
```

Comment:
Thanks for adding the different scenarios. What's missing is still a `file://` and a COOP URL. But those can be done later.

Reply:
Added a file:// test (see last commit). Not sure how to add COOP

Reply:
For COOP it could look like https://github.com/web-platform-tests/wpt/blob/master/webdriver/tests/classic/navigate_to/navigate.py#L42-L53. But we can really do that later. Also it probably will not make a change at all given that we most likely will not replace the browsing context group for a reload.